CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# v0.3.0
+
+## Changes
```

Comment:
The convention in rust-vmm is to use Changed, Fixed.. instead of Changes, Fixes.